scripts/install

```diff
@@ -67,6 +67,11 @@ then
     YNH_PHP_VERSION="$phpversion"
 fi
 
+# Add the custom error build with the application
+ynh_add_config --template="nginx-code-error.conf" --destination="$nginx_extra_conf_dir/error-code.conf"
+ynh_app_setting_set --app=$app --key=custom_error_file --value="true"
```

Comment:
If it's enabled by default but no actual `error/404.html` page exists, ain't this going to trigger an error inside the error ..? Or if it doesn't, why make this an option in the first place rather than always enabling it ...?